packages/app-collective/src/Overview/index.tsx

```diff
@@ -2,20 +2,75 @@
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
 
+import { AccountId, BlockNumber } from '@polkadot/types';
+import { ElectionsInfo } from './types';
+
+import BN from 'bn.js';
 import React from 'react';
 
+import { withCalls } from '@polkadot/ui-api';
+
 import Members from './Members';
+import SubmitCandidacy from './SubmitCandidacy';
 import Summary from './Summary';
 
-interface Props {}
+interface Props {
+  elections_members?: [string, BlockNumber][];
+  elections_candidates?: string[];
+  elections_candidateCount?: BN;
+  elections_desiredSeats?: BN;
+  elections_termDuration?: BN;
+  elections_voteCount?: BN;
+}
+
+interface State {
+  electionsInfo: ElectionsInfo;
+}
+
+class Overview extends React.PureComponent<Props, State> {
+  public static getDerivedStateFromProps ({
+    elections_members: members = [],
```

Comment:
No sure about the `elections_members: members = []` form, since props has the types, it should just be `elections_members = []`

Reply:
Just renames it to be put into electionsInfo :)

Reply:
omg... I dunno what I was smoking when I made that remark.